### DISCUSSION TITLE
Exit partial volt template

### DISCUSSION THREAD

**Discussion by crazylem** [phalcon/cphalcon#15398] [Q&A] [answered]
Hi
Is there a more elegant solution to terminate the partial template than `<?php return ?>`?
`partial.volt:`
```volt
{% if not isTemplateCompilable %}
  <?php return ?>
{% endif %}
// template body
// with lot of indents
// don't want to add one more
```

**Comment by Jeckerson** [comment, marked as answer]
Hello, no. Try to review your approach of templates. For example, by using `extends` + `blocks`, which will give you the possibility to only fill blocks that you want.

**Reply by crazylem** [reply]
Thank you.
